scripts/combine/setupCombine.py

```diff
@@ -193,6 +193,20 @@ def setup(args, inputFile, fitvar, xnorm=False):
     if args.fitXsec:
         datagroups.unconstrainedProcesses.append(base_group)
 
+    if "lumi" in fitvar:
+        # in case fit is split by runs/ cumulated lumi
+        # lumi axis only exists for data, add it for MC, and scale the MC according to the luminosity fractions
+        lumis = np.array([0.0, 0.419, 2.332, 4.329, 6.247, 8.072, 10.152, 12.265, 14.067, 15.994, 16.812])
```

Comment:
Better to define this in a common place together with the lumi axis so that the correspondence between lumi fractions and run ranges is more clear.